parse_mem.py

```diff
@@ -157,30 +96,50 @@ def create_rsa(self, output):
             univ.Integer(e1),
             univ.Integer(e2),
             univ.Integer(c),
-            )
+        )
 
-        struct = univ.Sequence()
+        s = univ.Sequence()
 
-        for i in xrange(len(seq)):
-            struct.setComponentByPosition(i, seq[i])
+        for i in range(len(seq)):
+            s.setComponentByPosition(i, seq[i])
 
-        raw = encoder.encode(struct)
+        raw = encoder.encode(s)
         data = base64.b64encode(raw)
 
         # chop data up into lines of certain width
         width = 64
-        chopped = [data[i:i + width] for i in xrange(0, len(data), width)]
+        chopped = [data[i:i + width].decode('utf-8') for i in range(0, len(data), width)]
```

Comment:
Is it safe to assume that this content will be UTF-8 encoded? Wouldn't it matter on the OS's preferred encoding on the server?